backend/src/main/scala/cromwell/backend/async/AsyncBackendJobExecutionActor.scala

```diff
@@ -77,6 +80,14 @@ trait AsyncBackendJobExecutionActor { this: Actor with ActorLogging =>
     case Finish(SuccessfulExecutionHandle(outputs, returnCode, hash, resultsClonedFrom)) =>
       completionPromise.success(BackendJobExecutionSucceededResponse(jobDescriptor.key, outputs))
       context.stop(self)
+    case Finish(FailedNonRetryableExecutionHandle(throwable, returnCode)) =>
+      completionPromise.success(BackendJobExecutionFailedNonRetryableResponse(jobDescriptor.key, throwable, returnCode))
+      context.stop(self)
+    case Finish(FailedRetryableExecutionHandle(throwable, returnCode)) =>
+      completionPromise.success(BackendJobExecutionFailedRetryableResponse(jobDescriptor.key, throwable, returnCode))
+      context.stop(self)
+    case Finish(cromwell.backend.async.AbortedExecutionHandle) => ???
```

Comment:
This PR is not about abort, I just filled this out as a favor to whoever was going to pick up the abort ticket so they'd know this was a place where they'd need to add code.  Without adding code things just silently don't work.